mantle/kola/tests/misc/multipath.go

```diff
@@ -75,6 +75,7 @@ systemd:
       contents: |
         [Unit]
         Description=Mount /var/lib/containers
+        After=ostree-remount.service
```

Comment:
optional..  I know git blame exists but sometimes when you're intentionally doing something subtle that has a lot of rationale behind it it's good to say why.. We could just put a link to this PR:

```suggestion
        Description=Mount /var/lib/containers
        # after ostree-remout because
        # see https://github.com/coreos/coreos-assembler/pull/2457
        After=ostree-remount.service
```

or we could restate the problem in a comment. Either way.. Optional.